### PR TITLE
Front end design of pages

### LINKED ISSUE #12: UI: Complete Links in the Navbar
General

Create the Home, Log In and Sign Up pages and have the navbar redirect the user to the corresponding page.

### LINKED ISSUE #14: UI: Format the Sign Up Page
General

Format the Sign Up page's UI according to the design.

### LINKED ISSUE #15: UI: Format the Log In Page
General

Format the Log In page according to the appropriate design.

### LINKED ISSUE #26: Front-end: Create a Post Format
#9 #10 

Create a Post object/format that takes an image and a caption. Format the post according to the design (with a Like button for the future).

### PR DESCRIPTION
**Pull request by jongjonathan**
- Navbar linked with templates
- Design of post
- Creation of Login and Signup template
- Instagram grid gallery

closes #26
closes #15
closes #14
closes #12
#9
#13

### CODE REVIEW

**Comment by jongjonathan** [comment]
@LiamBurchellReyes 
use app.py from backend, this version just has the templates linked to the navbar which can be easily added

File: templates/main.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        <img src="static\images\img6.jpg" alt="car6">
+        <img src="static\images\img7.jpg" alt="car7">
+        <img src="static\images\img8.jpg" alt="car8">
+        <img src="static\images\img9.jpg" alt="car9">
+    </div>
+
+
+   
+
+
```
Comment (LiamBurchellReyes) [inline templates/main.html:66]: Spacing not necessary, I think? Minor.

**Review by LiamBurchellReyes — APPROVED** [review]
Looks good to me. One comment on extra spacing in main.html, but changing it is not necessary

Also, comment about app.py noted.

**Review by JosephMezza — COMMENTED** [review]
Looking good. Now that the front-end has been improved we can now link it with the back-end.

**Review by JosephMezza — APPROVED** [review]
It is ready to be merged

**Pull request merged**